nucypher-core/src/threshold_message_kit.rs

```diff
@@ -0,0 +1,109 @@
+use alloc::boxed::Box;
+use alloc::string::String;
+use alloc::vec::Vec;
+
+use ferveo::api::{Ciphertext, CiphertextHeader, SharedSecret};
+use ferveo::Error;
+use serde::{Deserialize, Serialize};
+
+use crate::access_control::AccessControlPolicy;
+use crate::versioning::{
+    messagepack_deserialize, messagepack_serialize, ProtocolObject, ProtocolObjectInner,
+};
+
+// TODO should this be in umbral?
```

Comment:
@derekpierre After our chat about `encrypt_for_dkg` yesterday I have considered pulling some of the related code into `ferveo`. So a similar notion crossed my mind. However, since neither of those are directly related to cryptographic concept in `umbral` or `ferveo`, I think these are best saved for `nucypher-core` where other protocol-specific concepts reside.

Reply:
This is comment that gets moved by cargo fmt.

It was meant for serde_bytes, which is in umbral, but probably should be in a more common place.

Reply:
This is a comment that gets moved by the linter.

It was meant for serde_bytes (used previously), that we use in `nucypher-core` but it is implemented in `umbral`. Feels like that should be in some common library, or moved into serde itself. I believe Bogdan was trying to get it in serde.

Reply:
Ah, I see. I also experienced these antics from `cargo fmt`